abci/client/local_client.go

```diff
@@ -70,6 +72,7 @@ func (app *localClient) callback(req *types.Request, res *types.Response) *ReqRe
 func newLocalReqRes(req *types.Request, res *types.Response) *ReqRes {
 	reqRes := NewReqRes(req)
 	reqRes.Response = res
+	reqRes.Done() // release waiters on response
```

Comment:
This is a fix to the local and unsync clients. Now we can do `reqRes.Wait()`.

Reply:
I don't see this change on unsync client... my understanding is that this function is also used by the unsync client. If I'm right, could you add a comment on top of the function saying the unsync client also uses it?

Reply:
Yes, `newLocalReqRes` is called in `unsync_local_client`. I will add a comment.